app/javascript/views/walletServices/braveConnection/BitflyerConnection.tsx

```diff
@@ -85,9 +100,9 @@ class BitflyerConnection extends React.Component<any, any> {
                         />
                     </div>
                 </div>
-                {!this.props.isPayable && (
-                    <VerifyButton verifyUrl={this.props.verifyUrl}>
-                        <FormattedMessage id="walletServices.bitflyer.notPayable" />
+                {hasProblem && (
+                    <VerifyButton verifyUrl={verifyUrl}>
```

Comment:
Did we want to fix the verify button issue here, or should that be a separate issue?

Reply:
I can fix it really quickly

Reply:
@jlbyrne patched